config/manager/manager.yaml

```diff
@@ -23,6 +22,7 @@ spec:
       labels:
         control-plane: controller-manager
         sidecar.istio.io/inject: "false"
+        app.kubernetes.io/component: istio-operator.kyma-project.io
```

Comment:
Added in kustomize

Reply:
Unfortunately the labels in the `default/kustomization.yaml`doesn't apply here by default. 
I think we can use `includeTemplates: true` ([documentation](https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/labels/)) to make that work.

Reply:
It's already added

Reply:
Sorry, I must have been blind. 😵‍💫

Reply:
fyi, Unfortunately, `kustomize`:
```
./bin/kustomize edit set label app.kubernetes.io/version:123
```
removes `includeSelectors: false` from `default/kustomization.yaml` for me:
```
labels:
- includeTemplates: true
  includeSelectors: false
```
results in:
```
labels:
- includeTemplates: true
```
after the edit, which is a no-go because we *need* `includeSelectors: false` because of the immutable `selector` requirement for the `Deployment`. tried it on the current `kustomize` version (4.5.5) and the latest (5.3.0)